discovery/file/file_test.go

```diff
@@ -358,9 +358,7 @@ func TestInvalidFile(t *testing.T) {
 
 			// Verify that we've received nothing.
 			time.Sleep(defaultWait)
-			if runner.lastReceive().After(now) {
-				t.Fatalf("unexpected targets received: %v", runner.targets())
-			}
+			require.False(t, runner.lastReceive().After(now), "Unexpected targets received.")
```

Comment:
Should we supply the unexpected targets that were received, like the old error used to?